src/VisualStudio/IntegrationTest/TestUtilities/Interop/AutomationRetry/IUnknown.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.Interop.AutomationRetry
+{
+    [ComImport]
+    [Guid("00000000-0000-0000-C000-000000000046")]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    internal interface IUnknown
+    {
```

Comment:
👍 Will comment primarily because the purpose of this isn't to make use of `IUnknown`, but rather as a way to force arbitrary `object` instances returned by other calls in the UI Automation APIs to detour through `AutomationRetryWrapper<T>.GetInterface` so it can be wrapped according to the type about to be used.